README.md

```diff
@@ -1,128 +1,216 @@
 Hermione
-===========
+========
+Hermione is the utility for integration testing of web pages using [WebdriverIO](http://webdriver.io/) and [Mocha](https://mochajs.org).
 
-## Установка
+<img src="hermione.png" align="right"/>
 
-```shell
-npm i hermione --save-dev
-```
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Why should you choose hermione?](#why-should-you-choose-hermione)
+  - [Easy to use](#easy-to-use)
+  - [Parallel test running](#parallel-test-running)
+  - [Extensibility](#extensibility)
+  - [Retries for failed tests](#retries-for-failed-tests)
+  - [Executing a separate test](#executing-a-separate-test)
+  - [Auto initilization and closing grid sessions](#auto-initilization-and-closing-grid-sessions)
+- [Prerequisites](#prerequisites)
+- [Quick start](#quick-start)
+- [.hermione.conf.js](#hermioneconfjs)
+  - [specs](#specs)
+  - [browsers](#browsers)
+  - [grid](#grid)
+  - [baseUrl](#baseurl)
+  - [timeout](#timeout)
+  - [waitTimeout](#waittimeout)
+  - [slow](#slow)
+  - [debug](#debug)
+  - [sessionsPerBrowser](#sessionsperbrowser)
+  - [retry](#retry)
+  - [plugins](#plugins)
+  - [mochaOpts](#mochaopts)
+  - [prepareBrowser](#preparebrowser)
+  - [prepareEnvironment](#prepareenvironment)
+- [CLI](#cli)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Why should you choose hermione?
+`Hermione` provides several features which `WebdriverIO` doesn't and makes testing process easier.
+
+### Easy to use
```

Comment:
I added refs to webdriver and mocha in the first sentence but you're right. It'd be great to add refs here as well.